script/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '2.1'
+
+services:
+  testrunner:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    image: eventq
+    container_name: testrunner
+    command: bash -c "./scripts/container_loop.sh"
+    depends_on:
+      - redis
+      - rabbitmq
+    volumes:
+      - ./container_loop.sh:/scripts/container_loop.sh
+      - ../:/src
+    environment:
+      - AWS_ACCESS_KEY_ID=mock_id
+      - AWS_SECRET_ACCESS_KEY=mock_password
+      - AWS_REGION=eu-west-1
+      - AWS_SQS_ENDPOINT=http://goaws:4576
+      - AWS_SNS_ENDPOINT=http://goaws:4575
+#    env_file:
+#      - ../.aws.env
+
+  rabbitmq:
+    image: rabbitmq:3.6.5
+    container_name: rabbitmq
+
+  redis:
+    image: redis:alpine
+    container_name: eventq_redis
+    ports:
+      - "6379:6379"
+#  goaws:
+#    image: pafortin/goaws
+#    container_name: goaws
+#    ports:
+#      - "4100:4100"
+  goaws:
+    image: localstack/localstack
```

Comment:
goaws and localstack both have issues that the rabbitmq service does not appear to have.  So in the interim of needing a queue system in docker that appears to work as designed will require we probably still support rabbitmq from a testing perspective.
Currently, localstack has an issue with creating creating duplicate SNS topics,  goaws has an issue with *not* incrementing the retry attribute, and both have concurrency problems.

Reply:
existing specs test the exponential backoff with retries, so we have to use `localstack`.  I left `goaws` commented out because it is an option and is faster.